Application.hs

```diff
@@ -162,22 +165,18 @@ develMain = develMainHelper getApplicationDev
 -- | The @main@ function for an executable running this site.
 appMain :: IO ()
 appMain = do
-    -- Get the settings from all relevant sources
-    settings <- withEnv $ loadAppSettingsArgs
-        -- fall back to compile-time values, set to [] to require values at runtime
-        [configSettingsYmlValue]
-
-        -- allow environment variables to override
-        useEnv
-
-    -- Generate the foundation from the settings
+    settings <- getAppSettings
```

Comment:
Using `loadAppSettingsArgs` makes Yesod swallow the args while figuring out the environment. I'm not sure if we're using these arguments in production.